tests/python_client/testcases/test_utility.py

```diff
@@ -241,7 +241,7 @@ def test_loading_progress_not_existed_collection_name(self):
         self.collection_wrap.construct_from_dataframe(c_name, df, primary_field=ct.default_int64_field_name)
         self.collection_wrap.create_index(ct.default_float_vec_field_name, index_params=ct.default_flat_index)
         self.collection_wrap.load()
-        error = {ct.err_code: 1, ct.err_msg: "describe collection failed: can't find collection"}
+        error = {ct.err_code: 4, ct.err_msg: "collection default:not_existed_name: collection not found"}
```

Comment:
what does this mean? collection name?

Reply:
Yes. In this case, `loading_progress("not_existed_name", check_task=CheckTasks.err_res, check_items=error)` the collection name "not_existed_name" is given.